test/harness/e2e/vm/vm_linux.go

```diff
@@ -299,3 +305,17 @@ func (v *VMInLibvirt) IsRunning() (exists bool, err error) {
 		return false, nil
 	}
 }
+
+func (v *VMInLibvirt) RunAndWaitForSSH() error {
```

Comment:
Not explicitly required for this patch. But when I expect that some tests will require running the VM without waiting to SSH.